test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -70,10 +70,23 @@ const (
 )
 
 var (
+	extentionPoints = []string{
```

Comment:
-1 on this. Either we discover the labels, or we specify the labels in metrics explicitly like:

- scheduler_framework_extension_point_duration_seconds[extension_point="Filter"]

Or, you can define the metric a map so it's easy to parse.

Reply:
updated to use map, btw, I have search in the "https://github.com/prometheus/common", and haven't found a parser which capable to parse a PQL compatible string, easier for me to use map here. :)